src/Examples/CSharp DotNetFramework/Program.cs

```diff
@@ -1,40 +1,18 @@
-﻿using libplctag;
-using System;
-using System.Net;
-using System.Threading;
+﻿using System;
 
-namespace CSharpDotNetFramework
+namespace CSharpDotNetCore
```

Comment:
@jkoplo this one is in a .NET framework project - was this intentional?

Reply:
Oops. I copied and pasted from the other one. There's probably a better way to just have one example project that switches platform with a compilation target (`AnyCPU-Core`) - I think I've seen that done before.

Reply:
Oh cool I haven't seen that before.